config.py

```diff
@@ -46,3 +46,5 @@
 TELCO_CODE_ANSWERED = 16
 DEFAULT_LANGUAGE_ID = 1
 MAX_RETRY_ATTEMPTS_FOR_LOGS = os.environ.get("MAX_RETRY_ATTEMPTS_FOR_LOGS", 3)
+
+GOOGLE_APPLICATION_CREDENTIALS = os.environ.get("GOOGLE_APPLICATION_CREDENTIALS")
```

Comment:
This is added because every time while run the server locally we get an error that:
```
google.auth.exceptions.DefaultCredentialsError: File big-query-credentials.json was not found
```